ecl/hqlcpp/hqlcerrors.hpp

```diff
@@ -170,7 +170,7 @@
 #define HQLERR_CouldNotFindDataset              4158
 #define HQLERR_CouldNotAnyDatasetX              4159
 #define HQLERR_NestedThorNodes                  4160
-#define HQLERR_MissingTransformAssignXX         4161
+#define HQLERR_MissingTransformAssignX          4161
```

Comment:
Is this and the changes below related to this issue?

Reply:
Yes, it was tracing a pointer value in one of the error messages, so I remove it.  XX indicated there were two arguments, so removed one from the constant name.